src/main/java/com/microsoft/aad/msal4j/IConfidentialClientApplication.java

```diff
@@ -1,54 +1,33 @@
-// Copyright (c) Microsoft Corporation.
-// All rights reserved.
-//
-// This code is licensed under the MIT License.
-//
-// Permission is hereby granted, free of charge, to any person obtaining a copy
-// of this software and associated documentation files(the "Software"), to deal
-// in the Software without restriction, including without limitation the rights
-// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
-// copies of the Software, and to permit persons to whom the Software is
-// furnished to do so, subject to the following conditions :
-//
-// The above copyright notice and this permission notice shall be included in
-// all copies or substantial portions of the Software.
-//
-// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
-// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-// THE SOFTWARE.
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
 
 package com.microsoft.aad.msal4j;
 
 import java.util.concurrent.CompletableFuture;
 
+/**
+ * Interface representing a confidential client application (Web App, Web API, Daemon App).
+ * Confidential client applications are trusted to safely store application secrets, and therefore
+ * can be used to acquire tokens in then name of either the application or an user.
+ * For details see https://aka.ms/msal4jclientapplications
+ */
 public interface IConfidentialClientApplication extends IClientApplicationBase {
 
     /**
-     * Acquires security token from the authority.
-     *
-     * @param parameters#scopes scopes of the access request
-     * @return A {@link CompletableFuture} object representing the
-     * {@link IAuthenticationResult} of the call. It contains Access
-     * Token and the Access Token's expiration time. Refresh Token
-     * property will be null for this overload.
+     * Acquires tokens from the authority configured in the application, for the confidential client
+     * itself (in the name of no user)
```

Comment:
"in the name of no user" sounds weird ... like it was said by Master Yoda from Start Wars ))